src/app/destinyTrackerApi/d2-bulkFetcher.ts

```diff
@@ -20,16 +20,16 @@ class D2BulkFetcher {
     this._reviewDataCache = reviewDataCache;
   }
 
-  _getBulkWeaponDataEndpointPost(itemList: DtrItem[], platformSelection: number) {
+  _getBulkWeaponDataEndpointPost(itemList: DtrItem[], platformSelection: number, mode: number) {
```

Comment:
It'd be nice to type `platformSelection` as `DestinyMembershipType` and `mode` as whatever it is